apps/spruce/src/pages/waterfall/WaterfallGrid.tsx

```diff
@@ -46,7 +46,12 @@ export const WaterfallGrid: React.FC<WaterfallGridProps> = ({
             version ? (
               <VersionLabel key={version.id} size="small" {...version} />
             ) : (
```

Comment:
If you calculate the inactive version count here, then you can pass the value into each prop (either the count or `hasError`) instead of having to both count and use the `.some()` function in two separate places